attendance/API_HR/c-hr.php

```diff
@@ -975,6 +975,43 @@ public static function getUserMonthAttendace($userid, $year, $month) {
         return $finalReturn;
     }
 
+    public static function getAllYearAttendance() {
```

Comment:
the functions which are directly called by API  append `API_`   to it ( see other function you will get to know)